magenta-lib/src/main/scala/magenta/artifact/ArtifactHelper.scala

```diff
@@ -0,0 +1,30 @@
+package magenta.artifact
+
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ListObjectsV2Request, S3ObjectSummary}
+
+import scala.collection.JavaConverters._
+
+object ArtifactHelper {
+  def listObjects(path: S3Location)(implicit s3Client: AmazonS3) = {
```

Comment:
This looks quite a lot like https://github.com/guardian/riff-raff/blob/master/riff-raff/app/ci/S3Location.scala#L25

Is it worth pulling them out. Also I'm not a fan of names of the form `XyHelper`